functions/src/test/firestoreRefs.test.ts

```diff
@@ -6,15 +6,18 @@ import Refs from '../firestoreRefs'
 
 describe('Firestore Refs', () => {
   it('Returns the Activities ref', async () => {
-    // @ts-ignore
     const res = await Refs(mocks.firestoreMock).activities().doc('default').get()
-    const data = res.data()
+    const data = res.data()!
     assert(data.title === 'Security Meetup')
   })
-  it('Returns the Users ref', async () => {
-    // @ts-ignore
-    const res = await Refs(mocks.firestoreMock).users().doc('bob').get()
-    const data = res.data()
+  it('Returns a user ref', async () => {
+    const res = await Refs(mocks.firestoreMock).public_user('bob').get()
```

Comment:
You'll have to change the way mocks work to reference `public_user` instead of `users`. You may just want to have them duplicate each other :P